contracts/hardhat.config.ts

```diff
@@ -108,6 +112,46 @@ const config: HardhatUserConfig = {
       url: 'https://polygon-rpc.com/',
       gasPrice: 33000000000, // 33 gwei
     },
+    rinkarby: {
+      accounts: {
+        count: 10,
+        initialIndex: 0,
+        mnemonic,
+        path: "m/44'/60'/0'/0",
+      },
```

Comment:
Might be nice to define the account config into a standalone const and reuse it for each network to make it more DRY, reduce copy/paste errors, and a bit more readable by shortening the file

Reply:
Good idea. Tracking in https://github.com/ScopeLift/umbra-protocol/issues/305 since I don't have time to do it right now, and I want to unblock t his PR.